logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/EnableLoggingUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.wire.kalium.logic.feature.user
+
+import com.wire.kalium.logic.configuration.UserConfigRepository
+
+
+class EnableLoggingUseCase(
```

Comment:
Could we extract an interface out of this UseCase ? It will be easy I think in the future for espresso testing in case we want to create Fakes.

You can see an example on `UploadUserAvatarUseCase`